src/client/Client.ts

```diff
@@ -163,12 +164,23 @@ class Bot extends Client {
 
     public scheduleMessages() {
         this.logger.info(`Scheduling messages...`);
+        let next_class_embed: EmbedPage;
+
+        if (event.summary.includes('[Virtual]')) {
+            next_class_embed = this.embeds.get('nextVirtualClass') as EmbedPage;
+        } else if (event.summary.includes('[Presencial]')) {
+            next_class_embed = this.embeds.get(
+                'nextFaceToFaceClass'
+            ) as EmbedPage;
+        } else {
+            this.logger.error(`Invalid class type.`);
+        }
 
         cron.schedule(
             '0 00 18 * 3,4,5,6 1,4',
             () => {
                 this.logger.info('Sending class reminder...');
-                this.embeds.get('nextClass')!.send();
+                next_class_embed;
```

Comment:
Si no estoy equivocado, no falta enviar el `next_class_embed`?
```suggestion
                next_class_embed.send();
```

Reply:
Correcto, bien visto